gradebook.cpp

```diff
@@ -7,9 +7,61 @@ grade_Book::grade_Book(){}
 
 grade_Book::grade_Book(std::vector<std::string> assignment_name_list, std::vector<std::string> type, std::vector<int> score_obtained, std::vector<int> max_score){}
        
-double grade_Book::getIndividualGrade(std::string assignment_name){return 0.0;}
-double grade_Book::getCategoryGrade(std::string assignment_type){return 0.0;}
-double grade_Book::getCourseGrade(int option){return 0.0;}
+double grade_Book::getIndividualGrade(std::string assignment_name){
+    //creates an idex of the passed in assignment name
+    int idx;
+    idx = getAssignmentIndex(assignment_name);
+
+
+    //goes to the score_obtained vector to obtain and return the score of that assignment
+    //score as an percentage
+    //double score = ( (score_obtainted[idx] / max_score[idx]) * 100);
+    double score = score_obtainted[idx];

```

Comment:
line 18 should be used instead of 19, line 19 has an issue if you get 40/120, it would output 40 when it should output 33.3